CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 2.13.1 (Unreleased)
```

Comment:
[TF doc](https://www.terraform.io/plugin/sdkv2/best-practices/versioning#version-headers) is a little vague on if `Unreleased` will be replaced by the release date or not. We can always replace it with a quick PR if necessary